web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -175,9 +175,6 @@ import {get_string_diff} from "../../src/util";
     if (this.fixed) {
       this.$container.css('position', 'fixed');
     }
-    // The naturalSearch option causes arrow keys to immediately
-    // update the search box with the underlying values from the
-    // search suggestions.
```

Comment:
This isn't relevant anymore with pills.

Reply:
Yeah probably making this a follow-up commit would simplify review; should be very easy to split with `git checkout` to make removing this the first commit, and then reordering it to the end.